services/src/main/java/org/apache/druid/cli/CoordinatorJettyServerInitializer.java

```diff
@@ -89,24 +88,17 @@ public void initialize(Server server, Injector injector)
   {
     final ServletContextHandler root = new ServletContextHandler(ServletContextHandler.SESSIONS);
     root.setInitParameter("org.eclipse.jetty.servlet.Default.dirAllowed", "false");
+    root.setInitParameter("org.eclipse.jetty.servlet.Default.redirectWelcome", "true");
+    root.setWelcomeFiles(new String[]{"index.html", "legacy-coordinator-console.html"});
```

Comment:
What is the `index.html` entry for? I don't see an `index.html` page in the project, as far as I can tell from docs, the paths in the welcome files list will be appended, in order, to any partial request path until it finds a static resource that matches.

Reply:
Oh, is `old-console`?

Reply:
Yeah, i see so that  `/old-console` will drop on `/old-console/index.html`, nevermind, makes sense 🙃

Reply:
yup!